.rhcicd/clowdapp-backend.yaml

```diff
@@ -231,8 +231,8 @@ objects:
         SELECT
           b.display_name AS "Bundle",
           a.display_name AS "Application",
-          et.display_name AS "Event type",
-          es.subscription_type AS "Subscription type",
+          et.display_name::TEXT AS "Event type",
```

Comment:
Why would this display name require a transformation to text and not the previous ones?

Reply:
Good question. You are right, let's be safe and double down on casting everything to text, except the counts...

Reply:
Yes, yesterday they showed us in the meeting that the counts are getting read properly... So I think it is safe for us to leave them like this.